Format.pm

```diff
@@ -127,6 +129,11 @@ positive representation of the number being passed to that function.
 C<format_number()> and C<format_price()> utilize this feature by
 calling C<format_negative()> if the number was less than 0.
 
+C<ROUND_OPTION> is only used by C<round()> or C<format_number()>.
+-1 or ROUND_FLOOR means floor, -2 or ROUND_ABS_FLOOR means absolute
+floor, 1 or ROUND_CEIL means ceil, 2 or ROUND_ABS_CEIL means absolute
+ceil and a 0 or ROUND_NORMAL stand for normal rounding.
```

Comment:
change 'stand for' to 'means'. Also add "see the methods abs_floor(), floor(), abs_ceil(), and ceil() for more information"